js/impress.js

```diff
@@ -2807,22 +2808,22 @@
  *    overwritten for individual steps and media.
  *
  *    Examples:
- *    - data-media-autostart="true" data-media-autostop="true": start media on enter, stop on
+ *    - data-media-autoplay="true" data-media-autostop="true": start media on enter, stop on
```

Comment:
`autostart` was changed automatically to `autoplay` after `npm run build`. I'm doing something wrong perhaps 🤔

Reply:
No, it's not you. A previous commit forgot to run build.js. It is fixed in HEAD now: https://github.com/impress/impress.js/commit/99484d11b855ebd8d5f726023ca79b88e5c42d18

You can ignore this, I think it should go away in merge. Alternatively, you can do a git merge from master to get rid of it.